src/nbl/asset/utils/CFilesystemIncluder.h

```diff
@@ -20,8 +20,8 @@ class CFilesystemIncluder : public IIncluder
 
     void addSearchDirectory(const std::string& _searchDir) override
     {
-        io::path absPath = m_filesystem->getAbsolutePath(_searchDir.c_str());
-        IIncluder::addSearchDirectory(absPath.c_str());
+        std::filesystem::path absPath = m_filesystem->getAbsolutePath(_searchDir.c_str());
```

Comment:
isnt there a `std::filesystem` utility for that?

Then we could nuke the `getAbsolutePath` method from the `IFileSystem` class

Reply:
yeah i think i just havend got there yet, but mb